src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -60,6 +60,14 @@ export class WinstonLoggerAdapter extends LoggerAdapter {
     return logger.error.apply(undefined, arguments);
   }
 
+  get logger() {
+    return logger;
+  }
+
+  configureLogger(options) {
+    configureLogger(options);
+  }
+
   addTransport(transport) {
     // Note that this is calling addTransport
     // from logger.  See import - confusing.
```

Comment:
Can we make this not confusing, e.g. call logger.addTransport() or similar instead?